src/domain/harbors/utils.ts

```diff
@@ -1,5 +1,10 @@
 import { HARBORS } from './__generated__/HARBORS';
 
+type Map = {
```

Comment:
We could extract these types to `types.ts`

Reply:
Done